src/views/ApartmentApp/components/StudentApplication/components/InstructionsCard/index.js

```diff
@@ -23,12 +24,11 @@ const InstructionsCard = () => {
 
   useEffect(() => {
     const getYear = () => setThisYear(new Date().getFullYear());
+    const loadApartmentSelectionDate = async () =>
+      setApartmentSelectionDate(await housing.getApartmentSelectionDate());
 
     getYear();
```

Comment:
```suggestion
    setThisYear(new Date().getFullYear());
    setApartmentSelectionDate(await housing.getApartmentSelectionDate();
```

Reply:
This caused a build error due to the use of `await` outside of an async function. 
Should the selection date be returned to a helper function, or it is possible to do something like 
```
useEffect(async () => {
  // the stuff inside
}, []);
```

Reply:
Oh right, I forgot about the needed `async` context. The callback passed to useEffect unfortunately cannot be async directly. The two workarounds are to define an async function and call it within the callback (which you were already doing), or use `Promise.then()` syntax. It's your call which you think is cleaner in this context.